exes/ec/tests/test_ec_util.cpp

```diff
@@ -62,4 +62,10 @@ auto main(int, char**) -> int {
     expect(exp.has_value() >> fatal);
     expect(exp.value() == test);
   };
+
+  "Test atv320 custom units"_test = []() {
+    static_assert(std::chrono::seconds{ 1 } == tfc::ec::devices::schneider::deciseconds{ 10 });
```

Comment:
do you wan't to have this here?

Reply:
For now yes, later I want to create a test just for the atv320.